coordinator/src/routes.rs

```diff
@@ -138,6 +142,8 @@ pub fn router(
             get(get_settings).put(update_settings),
         )
         .route("/api/admin/sync", post(post_sync))
+        .route("/api/admin/waitlist/add", post(add_to_waitlist))
```

Comment:
There is no need to add `add`. Just put both, the `post` and `get` onto `/api/admin/waitlist`